src/open_inwoner/userfeed/hooks/case_status.py

```diff
@@ -23,7 +23,8 @@ def case_status_notification_received(user: User, case: Zaak, status: Status):
         "case_uuid": case.uuid,
         "case_identificatie": case.identificatie,
         "case_omschrijving": case.omschrijving,
-        "status_omschrijving": status.statustype.omschrijving,
+        "status_omschrijving": status.statustype.statustekst
+        or status.statustype.omschrijving,
```

Comment:
For the future: these 'or' assignments statements in a dictionary will format a nudge better if you wrap with a parenthesis (visually more connected and less looking for subtle comma's).

Reply:
The parentheses are removed by black